extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeMixPlaylistExtractorTest.java

```diff
@@ -44,8 +41,7 @@
     private static final String VIDEO_TITLE =
             "Most Beautiful And Emotional  Piano: Anime Music Shigatsu wa Kimi no Uso OST IMO";
     private static final String RESOURCE_PATH = DownloaderFactory.RESOURCE_PATH + "services/youtube/extractor/mix/";
-    private static final Map<String, String> dummyCookie
-            = Collections.singletonMap(YoutubeMixPlaylistExtractor.COOKIE_NAME, "whatever");
```

Comment:
Whats the reason to replace this with `dummyCookie.put(YoutubeMixPlaylistExtractor.COOKIE_NAME, "whatever");` which needs to be called in every test class, instead of just once here?

Reply:
nevermind, i just saw the commit message